build.gradle

```diff
@@ -324,7 +324,7 @@ subprojects {
 
             check.dependsOn("testModules")
 
-            if (!(project.name in ['micrometer-commons', 'micrometer-observation', 'micrometer-observation-conventions', 'micrometer-observation-test'])) {
+            if (!(project.name in ['micrometer-commons', 'micrometer-observation', 'micrometer-observation-conventions', 'micrometer-observation-test', 'micrometer-jetty11'])) {
```

Comment:
We should make an issue for 1.11 to remove these. Otherwise, I think it will be easy to forget.

Reply:
fyi: https://github.com/micrometer-metrics/micrometer/issues/3486